feat: Add new new push notification interface `ParseNotification` for managing push notifications (#949)
parse-community/Parse-SDK-Flutter: packages/flutter/CHANGELOG.md

## [6.0.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-5.1.2...flutter-6.0.0) (2023-08-06)

### BREAKING CHANGES

> * Add new new push notification interface `ParseNotification` for managing push notifications. ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))

Reviewer: ```suggestion
* The push notification library flutter_local_notifications is replaced with the new push notification interface ParseNotification ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))
```

> * The push notification library flutter_local_notifications is replaced with the new push notification interface ParseNotification ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))
> 
> ### Features
> 
> * Add new new push notification interface ParseNotification for managing push notifications ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))

Reviewer: ```suggestion
* The push notification library flutter_local_notifications is replaced with the new push notification interface `ParseNotification` ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))

### Features

* Add new new push notification interface `ParseNotification` for managing push notifications ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))
```

> ## [6.0.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-5.1.2...flutter-6.0.0) (2023-08-02)

Reviewer: ```suggestion
## [6.0.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-5.1.2...flutter-6.0.0) (2023-08-06)
```

### BREAKING CHANGES

* Add new new push notification interface `ParseNotification` for managing push notifications. ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))

> 

Reviewer: ```suggestion

### Features

* Add new new push notification interface ParseNotification for managing push notifications ([#949](https://github.com/parse-community/Parse-SDK-Flutter/pull/949))

```
